src/styles/style.css

```diff
@@ -0,0 +1,127 @@
+:root {
+  --main-color: #3d4e61;
+}
+
+/* #region font */
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Book.ttf') format('ttf');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir, inherit;
```

Comment:
The `font-family` property should not include 'inherit'. It should only specify the font name, like `font-family: Avenir;`. The use of 'inherit' here is incorrect and may cause unexpected behavior.